jena-arq/src/main/java/org/apache/jena/sparql/engine/iterator/QueryIterProjectMerge.java

```diff
@@ -37,68 +37,69 @@
  * should not be used.
  */
 public class QueryIterProjectMerge extends QueryIterRepeatApply {
-    private final OpProject  opProject ;
-    private final OpExecutor engine ;
+    private final OpProject  opProject;
+    private final OpExecutor engine;
 
     public QueryIterProjectMerge(OpProject opProject, QueryIterator input, OpExecutor engine, ExecutionContext execCxt) {
-        super(input, execCxt) ;
-        this.opProject = opProject ;
-        this.engine = engine ;
+        super(input, execCxt);
+        this.opProject = opProject;
+        this.engine = engine;
     }
 
     @Override
     protected QueryIterator nextStage(Binding binding) {
         QueryIterator qIter = engine.executeOp(opProject.getSubOp(),
-                                               QueryIterSingleton.create(binding, getExecContext())) ;
+                                               QueryIterSingleton.create(binding, getExecContext()));
         qIter = new QueryIterConvert(qIter,
                                      new ProjectEnsureBindingConverter(binding, opProject.getVars()),
-                                     getExecContext()) ;
-        return qIter ;
+                                     getExecContext());
+        return qIter;
     }
 
     /** Ensure binding from outer binding are present when projecting */
     static class ProjectEnsureBindingConverter implements QueryIterConvert.Converter {
 
-        private final Binding   outerBinding ;
-        private final List<Var> projectionVars ;
+        private final Binding   outerBinding;
+        private final List<Var> projectionVars;
 
         public ProjectEnsureBindingConverter(Binding outerBinding, List<Var> vars) {
-            this.outerBinding = outerBinding ;
-            this.projectionVars = vars ;
+            this.outerBinding = outerBinding;
+            this.projectionVars = vars;
         }
 
         @Override
         public Binding convert(Binding bind) {
-            return ensure(projectionVars, outerBinding, bind) ;
+            return ensure(projectionVars, outerBinding, bind);
             // Effectively, this is:
-            //bind = new BindingProject(projectionVars, bind) ;
-            // return BindingUtils.merge(outerBinding, bind) ;
+            //bind = new BindingProject(projectionVars, bind);
+            // return BindingUtils.merge(outerBinding, bind);
         }
 
-        /** Merge two bindings, the outer and inner, projecting the inner with the give variables.
-         *  This is what happens in substitutIon execution, with a inner select-project.
+        /**
+         * Merge two bindings, the outer and inner, projecting the inner with the give variables.
```

Comment:
s/give/given but it's not in this PR, so we can fix it later...

Reply:
Done!